weblate/machinery/tests.py

```diff
@@ -378,14 +380,44 @@ def test_microsoft_cognitive(self):
         )
         responses.add(
             responses.GET,
-            'https://api.microsofttranslator.com/V2/Ajax.svc/'
-            'GetLanguagesForTranslate',
+            'https://api.cognitive.microsofttranslator.com/'
+            'languages?api-version=3.0',
             json=["en", "cs"],
```

Comment:
This doesn't match at all what is documented response here: https://docs.microsoft.com/en-us/azure/cognitive-services/translator/reference/v3-0-languages#response-body

That makes the service not working, because when executed with real service, the list of supported languages becomes: `{'dictionary', 'transliteration', 'translation'}`

Reply:
I also do not need to fetch supported languages in both tests, do I?